test/apps/iodemo/io_demo.cc

```diff
@@ -1410,6 +1409,42 @@ class DemoClient : public P2pDemoCommon {
         return data_size;
     }
 
+    static void
+    get_server_info_uncompleted_ops(const server_info_t& server_info,
+                                    std::stringstream &ss)
+    {
+        ss << get_num_uncompleted(server_info) << " uncompleted operations"
+           << " (read: " << server_info.num_completed[IO_READ] << "/"
+           << server_info.num_sent[IO_READ] << "; write: "
```

Comment:
can remove extra words, `; ` and `()` from the print, like this:
`server[5] read 5/10 write 10/23 (disconnecting)`
`server[15] read 8/10 write 10/29`